### ISSUE TITLE
[Bug?]: `start.server.preset` does not change the Nitro build target `preset`

### ISSUE THREAD

**Issue by charkour** [solidjs/solid-start#1208]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

`node-server` is used as the Nitro target when running `vinxi build` even when I specify this in `vite.config.ts`:

```ts
import { defineConfig } from "@solidjs/start/config";

export default defineConfig({
  start: {
    server: {
      preset: "netlify" // this has no effect
    }
  }
});
```

### Expected behavior 🤔

When I run the `vinxi build` command, I'd expect it to use the Nitro build target specified in `start.server.preset` following the Solid Start docs [here](https://start.solidjs.com/api/vite#configuring-your-application). Thank you!

### Steps to reproduce 🕹

Steps:

1. Create a new project with `pnpm create solid`
2. update the `vite.config.ts` to match the Solid Start [docs](https://start.solidjs.com/api/vite) using `netlify` as the `preset`. 
3. Run `pnpm build` and inspect the output (Nitro will use `node-server` instead of `netlify`).

### Context 🔦

I'm trying to update my old Solid Start project to use Netlify. I had to [patch](https://github.com/charkour/charlie.social/blob/main/patches/vinxi%400.0.62.patch) `vinxi` to get this to work.

**Comment by charkour** [comment]
Here is [related discussion](https://discord.com/channels/722131463138705510/910635844119982080/1191241521245192253) on Discord. Looks like this may be an issue from https://github.com/nksaraf/vinxi due to a recent commit.

**Comment by edivados** [comment]
Using the cli arg `--preset` should work as a workaround until fixed. No need to patch.
`vinxi build --preset netlify`

**Comment by charkour** [comment]
Thanks @edivados !

**Comment by asterikx** [comment]
The issue seems to be have been fixed in `vinxi@0.0.64`. I no longer need to add the cli arg `--preset netlify` or set the env var `NITRO_PRESET=netlify`.

**Comment by nksaraf** [comment]
Yup fixed with vinxi 0.0.64

**Comment by charkour** [comment]
Thank you!
